torch/csrc/jit/symbolic_script.cpp

```diff
@@ -383,6 +383,17 @@ const std::vector<std::string> functions = {
             return torch.matmul(self, other), backward
     )",
     R"(
+        def addcmul(self,
+                    tensor1,
+                    tensor2,
+                    *,
+                    value: float = 1.0):
+            def backward(grad_output):
+                grad_tensor1 = (grad_output * tensor2 * value)._grad_sum_to_size(tensor1.size())
```

Comment:
for non-fused cases you can make it slightly more efficient by computing `grad_output * value` intermediate instead of recomputing it for `grad_tensor1` and `grad_tensor2`.